packages/sdk/src/next/types/VariableTypes.ts

```diff
@@ -0,0 +1,6 @@
+import { ListVariableItem, Variable } from '../../types/VariableTypes';
+
+export interface ListVariable extends Variable {
```

Comment:
How will this now behave when importing the `ListVariable` type?

Shouldn't we also have a `src/next/index.ts`?

Reply:
I'm going to investigate our options